lib/Doctrine/DBAL/Portability/Connection.php

```diff
@@ -23,6 +23,10 @@ class Connection extends \Doctrine\DBAL\Connection
     public const PORTABILITY_EMPTY_TO_NULL = 4;
     public const PORTABILITY_FIX_CASE      = 8;
 
+    /**#@+
```

Comment:
First time I see this notation… where can I read more about it?

Reply:
See https://github.com/doctrine/dbal/pull/3795#issuecomment-568921124.

Reply:
TIL